modules/zivid/_calibration/detector.py

```diff
@@ -0,0 +1,40 @@
+# pylint: disable=missing-module-docstring
```

Comment:
The reasoning was that these modules in _calibration should never be imported by a customer (they are imported into zivid.calibration by us).
Do you think that's a good enough reason, or should I write a docstring?

Reply:
Would it not be possible to globaly disable `D107` and `missing-module-docstring` for all files in `_*` dirs? Then files could be kept simple.

Reply:
Maybe. But honestly I could just make docstrings for these modules explaining what I wrote above. I does no harm, and it would reduce confusion for future developers.

Reply:
Added docstrings to all these modules. All "# pylint: disable=missing-module-docstring" are now gone from the repo.